src/packages/templating/stylesheets/components/stylesheet-rule-input/stylesheet-rule-input.element.ts

```diff
@@ -32,18 +32,24 @@ export class UmbStylesheetRuleInputElement extends FormControlMixin(UmbLitElemen
 	}
 
 	#appendRule = async () => {
-		const { rule: newRule } = await this.#openRuleSettings(null);
-		if (!newRule) return;
-		this.rules = [...this.rules, newRule];
-		this.dispatchEvent(new UmbChangeEvent());
+		this.#openRuleSettings(null)
+			.then((value) => {
+				if (!value.rule) return;
+				this.rules = [...this.rules, value.rule];
+				this.dispatchEvent(new UmbChangeEvent());
+			})
+			.catch(() => undefined);
 	};
 
 	#editRule = async (rule: UmbStylesheetRule, index: number) => {
```

Comment:
This doesn't need to be async anymore:

```suggestion
	#editRule = (rule: UmbStylesheetRule, index: number) => {
```